src/Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -467,18 +482,23 @@ internal static SKImage ResizeImage(SKBitmap source, SKImageInfo targetInfo, boo
     {
         using var surface = SKSurface.Create(targetInfo);
         using var canvas = surface.Canvas;
-        using var paint = new SKPaint
-        {
-            FilterQuality = SKFilterQuality.High,
-            IsAntialias = isAntialias,
-            IsDither = isDither
-        };
+        using var paint = new SKPaint();
+        paint.IsAntialias = isAntialias;
+        paint.IsDither = isDither;
+
+        // Historically, kHigh implied cubic filtering, but only when upsampling.
+        // If specified kHigh, and were down-sampling, Skia used switch back to kMedium (bilinear filtering plus mipmaps).
```

Comment:
```suggestion
        // If specified kHigh, and were down-sampling, Skia used to switch back to kMedium (bilinear filtering plus mipmaps).
```
Not sure if that us what you meant?